bootstrap/shoppingmall_seed.py

```
@@ -0,0 +1,851 @@
+    # 42개 상품에 대해 안정적인 가격 분포를 부여한다.
+    base = [3200, 5500, 8900, 12000, 18900, 24000, 32000, 45000, 52000]
+    return base[(pid - 1) % len(base)]
+
+
+def _product_discount(pid: int) -> int:
+    values = [0, 5, 10, 15, 20]
+    return values[(pid - 1) % len(values)]
+
+
+def seed_core_data(db) -> SeedState:
+    now = datetime.utcnow()
```

Ruff (0.15.10)

[warning] `datetime.datetime.utcnow()` used

(DTZ003)